sub/project_settings/{{ cookiecutter.__folder_name }}/backend/scripts/create_site.py

```diff
@@ -27,7 +27,9 @@ def asbool(s):
 
 
 DELETE_EXISTING = asbool(os.getenv("DELETE_EXISTING"))
+{% if cookiecutter.__feature_headless == "1" -%}
```

Comment:
I don't see why a ClassicUI project won't make use of having Example content.

Reply:
Or we replace it with distributions (https://github.com/plone/cookieplone-templates/issues/114)

Reply:
Let's postpone that until #114 and for classic drop the broken example content for now.